src/goto-symex/reachability_tree.h

```diff
@@ -378,6 +378,11 @@ class reachability_treet
   /** Are we using the --schedule scheduling method? */
   bool schedule;
 
+  /* data structures to store global variables*/
+  std::map<expr2tc, std::list<unsigned int>> vars_map;
+  /* associative container that contains global writes in */
+  std::map<expr2tc, bool> is_global;
+
```

Comment:
Do you need ordering here? Otherwise, this should be an `unordered_map`.

Reply:
If it is only used to track a boolean, why use a map at all? An `unordered_set` should be better.

Reply:
I have declared `vars_map` as `unordered_map` and `is_global` as `unordered_set`.